<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestVpr</class>
 <widget class="QWidget" name="TestVpr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>85</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>85</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>84</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>85</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>车牌图片：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="QLabel" name="labPictrue">
     <property name="text">
      <string>车牌图片</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>车牌号：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLabel" name="labPlate">
     <property name="text">
      <string>车牌号</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5" colspan="2">
    <widget class="QLabel" name="labcolor">
     <property name="text">
      <string>车牌颜色</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="7">
    <widget class="QLabel" name="labmessage">
     <property name="text">
      <string>显示信息</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>车牌颜色：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="pushButton_OpenDev">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>打开设备</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="pushButton_CloseDev">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>关闭设备</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="pushButton_Capture">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>抓拍</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
